RELEASE_PROCESS.md

```diff
@@ -88,36 +88,21 @@ release.
 Before uploading, check to see the currently available versions:
 
     charm2 info --include=id,perm cs:~yellow/juju-gui
-    charm2 info --include=id,perm cs:~yellow/development/juju-gui
 
 Next, to upload the charm, go to the charm source directory and do:
 
-    make clean-tests
-    # NB: trusty is the default series unless specified in metadata.yaml.
-    charm2 upload . cs:~yellow/trusty/juju-gui
+    make publish-yellow
```

Comment:
the `make clean-tests` is removed. it was incorrect.  clean is better because it removed the .pyc files.